### DISCUSSION TITLE
What are the colors used for code examples on the Gintronic homepage

### DISCUSSION THREAD

**Discussion by ThorstenRhau** [Mark2Mark/gintronic#16] [General]
I would like to learn what the color scheme is called that is used on the Gintronic home page for code examples.

**Comment by Mark2Mark** [comment]
Thanks for your question.

Unfortunately this is not an existing theme, I made it up for the specimen in Indesign and never found the time to make it an actual theme. I am still wanting to do that, but as I also use VSCode and find it particularly cumbersome to make a theme for VSC, this never happened yet, but maybe at some point.

Hope that helps?

**Reply by ThorstenRhau** [reply]
Thank you for your quick reply.
Sad that this has not become a theme. Especially the light variant is gorgeous. It is light but not to light, has contrast but not to much or to little etc. 

Yes, it helps to know that I can stop searching for the theme that I have never seen before ;-)
